config/runtime.exs

```diff
@@ -255,7 +255,7 @@ end
 # OIDC Auth
 auth_oidc_env = System.get_env("AUTH_OIDC")
 
-if auth_oidc_env do
+if config_env() != :test && auth_oidc_env do
```

Comment:
Does test env have this environment variable `AUTH_OIDC`?

Reply:
Yeah, `.env` is env-agnostic (at least for me). Maybe we should clear these env vars for the `test` env, or update `runtime.exs` to be env-var-agnostic when `config_env() == :test`? Would prevent other env vars from causing bugs in test.